### ISSUE TITLE
inlcude_tasks in molecule tests fails to lint

### ISSUE THREAD

**Issue by joshtrutwin** [ansible/ansible-lint#3674]
##### Summary

If I set up a "molecule verify" with a verify.yml that uses include_tasks in a shared folder, ansible-lint is reporting:

```
syntax-check[specific]: 'XYZ' is not a valid attribute for a Play
molecule/shared/verify-common.yml:3:3
```

where XYZ is the first task name in the included file.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
ansible-lint 6.17.2 using ansible-core:2.15.1 ansible-compat:4.1.2 ruamel-yaml:None ruamel-yaml-clib:None
```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

ansible and ansible-lint installed via pip as non-root user on Amazon Linux.

##### STEPS TO REPRODUCE

create the following file structure:

base/ansible/molecule/default/verify.yml
base/ansible/molecule/shared/verify-common.yml

verify.yml is a playbook with basic ansible header and an include_tasks to verify-common.yml

```
---
- name: "Verify test fixture."
  hosts: all
  gather_facts: true
  any_errors_fatal: true

  tasks:
    - name: "Load common verification tasks"
      include_tasks: ../shared/verify-common.yml
```

verify-common.yml just has a basic task:

```
---

- name: Set a variable
  set_fact:
    hello: "world"
```

```console (paste below)
cd base/ansible
ansible-lint
```

in case above reports:

``` 
syntax-check[specific]: 'set_fact' is not a valid attribute for a Play
molecule/shared/verify-common.yml:3:3
```

##### Desired Behavior

ansible-lint should validate this file as a basic .yml file.  Looking at file_utils.py kind_from_path() function it is using the config "DEFAULT_KIND" pattern ```"**/molecule/*/*.{yaml,yml}"``` to interpret this file as a "playbook" kind.

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behavior

Please give some details of what is happening.
Include a [minimum complete verifiable example] with:

- minimized playbook to reproduce the error
- the output of running ansible-lint including the command line used
- if you're getting a stack trace, also the output of
  `ansible-playbook --syntax-check playbook`

```
$ ansible-lint -vv molecule/shared/verify-common.yml
DEBUG    Logging initialized to level 10
INFO     Identified /home/gaction/projects/packer-github-runner-ami/ansible as project root due config file /home/gaction/projects/packer-github-runner-ami/ansible/.ansible-lint.
DEBUG    Options: Options(cache_dir=PosixPath('/home/gaction/.cache/ansible-compat/0cb87f'), colored=True, configured=True, cwd=PosixPath('/home/gaction/projects/packer-github-runner-ami/ansible'), display_relative_path=True, exclude_paths=['.cache', '.git', '.hg', '.svn', '.tox'], format=None, lintables=['molecule/shared/verify-common.yml'], list_rules=False, list_tags=False, write_list=[], parseable=False, quiet=False, rulesdirs=[PosixPath('/home/gaction/.local/lib/python3.9/site-packages/ansiblelint/rules')], skip_list=[], tags=[], verbosity=2, warn_list=['no-changed-when', 'package-latest', 'fqcn', 'ignore-errors', 'name'], mock_filters=[], mock_modules=[], mock_roles=[], loop_var_prefix=None, only_builtins_allow_collections=[], only_builtins_allow_modules=[], var_naming_pattern=None, offline=False, project_dir='.', extra_vars=None, enable_list=[], skip_action_validation=True, strict=False, rules={}, profile=None, task_name_prefix='{stem} | ', sarif_file=None, config_file='/home/gaction/projects/packer-github-runner-ami/ansible/.ansible-lint', generate_ignore=False, rulesdir=[], use_default_rules=False, version=False, list_profiles=False, ignore_file=None)
DEBUG    CWD: /home/gaction/projects/packer-github-runner-ami/ansible
INFO     Running ansible-galaxy role install -vr roles/requirements.yml --roles-path /home/gaction/.cache/ansible-compat/0cb87f/roles
INFO     Set ANSIBLE_LIBRARY=/home/gaction/.cache/ansible-compat/0cb87f/modules:/home/gaction/.ansible/plugins/modules:/usr/share/ansible/plugins/modules
INFO     Set ANSIBLE_COLLECTIONS_PATH=/home/gaction/.cache/ansible-compat/0cb87f/collections:/home/gaction/.ansible/collections:/usr/share/ansible/collections
INFO     Set ANSIBLE_ROLES_PATH=/home/gaction/.cache/ansible-compat/0cb87f/roles:roles:/home/gaction/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
DEBUG    Loading custom .yamllint.yml config file, this extends our internal yamllint config.
DEBUG    Effective yamllint rules used: {'anchors': {'level': 'error', 'forbid-undeclared-aliases': True, 'forbid-duplicated-anchors': False, 'forbid-unused-anchors': False}, 'braces': {'level': 'error', 'forbid': False, 'min-spaces-inside': 0, 'max-spaces-inside': 1, 'min-spaces-inside-empty': -1, 'max-spaces-inside-empty': -1}, 'brackets': {'level': 'error', 'forbid': False, 'min-spaces-inside': 0, 'max-spaces-inside': 1, 'min-spaces-inside-empty': -1, 'max-spaces-inside-empty': -1}, 'colons': {'level': 'error', 'max-spaces-before': 0, 'max-spaces-after': -1}, 'commas': {'level': 'error', 'max-spaces-before': 0, 'min-spaces-after': 1, 'max-spaces-after': -1}, 'comments': False, 'comments-indentation': False, 'document-end': False, 'document-start': False, 'empty-lines': {'level': 'error', 'max': 3, 'max-start': 0, 'max-end': 0}, 'empty-values': False, 'float-values': False, 'hyphens': {'level': 'error', 'max-spaces-after': 1}, 'indentation': False, 'key-duplicates': {'level': 'error'}, 'key-ordering': False, 'line-length': False, 'new-line-at-end-of-file': False, 'new-lines': {'level': 'error', 'type': 'unix'}, 'octal-values': False, 'quoted-strings': False, 'trailing-spaces': False, 'truthy': False}
WARNING  Skipped installing old role dependencies due to running in offline mode.
INFO     Set ANSIBLE_LIBRARY=/home/gaction/.cache/ansible-compat/0cb87f/modules:/home/gaction/.ansible/plugins/modules:/usr/share/ansible/plugins/modules
INFO     Set ANSIBLE_COLLECTIONS_PATH=/home/gaction/.cache/ansible-compat/0cb87f/collections:/home/gaction/.ansible/collections:/usr/share/ansible/collections
INFO     Set ANSIBLE_ROLES_PATH=/home/gaction/.cache/ansible-compat/0cb87f/roles:roles:/home/gaction/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
INFO     Executing syntax check on playbook molecule/shared/verify-common.yml (1.04s)
WARNING  Listing 1 violation(s) that are fatal
syntax-check[specific]: 'set_fact' is not a valid attribute for a Play
molecule/shared/verify-common.yml:3:3


DEBUG    Determined rule-profile order: {'internal-error': (0, 'min'), 'load-failure': (1, 'min'), 'parser-error': (2, 'min'), 'syntax-check': (3, 'min'), 'command-instead-of-module': (4, 'basic'), 'command-instead-of-shell': (5, 'basic'), 'deprecated-bare-vars': (6, 'basic'), 'deprecated-local-action': (7, 'basic'), 'deprecated-module': (8, 'basic'), 'inline-env-var': (9, 'basic'), 'key-order': (10, 'basic'), 'literal-compare': (11, 'basic'), 'jinja': (12, 'basic'), 'no-free-form': (13, 'basic'), 'no-jinja-when': (14, 'basic'), 'no-tabs': (15, 'basic'), 'partial-become': (16, 'basic'), 'playbook-extension': (17, 'basic'), 'role-name': (18, 'basic'), 'schema': (19, 'basic'), 'name': (20, 'basic'), 'var-naming': (21, 'basic'), 'yaml': (22, 'basic'), 'name': (23, 'moderate'), 'name': (24, 'moderate'), 'name': (25, 'moderate'), 'spell-var-name': (26, 'moderate'), 'avoid-implicit': (27, 'safety'), 'latest': (28, 'safety'), 'package-latest': (29, 'safety'), 'risky-file-permissions': (30, 'safety'), 'risky-octal': (31, 'safety'), 'risky-shell-pipe': (32, 'safety'), 'galaxy': (33, 'shared'), 'ignore-errors': (34, 'shared'), 'layout': (35, 'shared'), 'meta-incorrect': (36, 'shared'), 'meta-no-tags': (37, 'shared'), 'meta-video-links': (38, 'shared'), 'meta-version': (39, 'shared'), 'meta-runtime': (40, 'shared'), 'no-changed-when': (41, 'shared'), 'no-changelog': (42, 'shared'), 'no-handler': (43, 'shared'), 'no-relative-paths': (44, 'shared'), 'max-block-depth': (45, 'shared'), 'max-tasks': (46, 'shared'), 'unsafe-loop': (47, 'shared'), 'avoid-dot-notation': (48, 'production'), 'sanity': (49, 'production'), 'fqcn': (50, 'production'), 'import-task-no-when': (51, 'production'), 'meta-no-dependencies': (52, 'production'), 'single-entry-point': (53, 'production'), 'use-loop': (54, 'production')}
                  Rule Violation Summary
 count tag                    profile rule associated tags
     1 syntax-check[specific] min     core, unskippable

Failed: 1 failure(s), 0 warning(s) on 1 files.

```

ansible-playbook --syntax-check is not valid for this scenario as this is not a playbook, just a file containing tasks.

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by joshtrutwin** [comment]
I don't know how serious this issue is as I can trick it out by moving my include into another sub-folder:

doesn't work: include_tasks: ../shared/verify-common.yml
works: include_tasks: ../shared/includes/verify-common.yml

**Comment by ssbarnea** [comment]
Put it inside a folder names `tasks` and you will be fine. At this moment we do not have any plans to do something about this because is already a bad code layout example.

Tasks files show never be outside `tasks` folders, just because it confuses people.

**Comment by joshtrutwin** [comment]
Sounds good, thanks for the info.
